pom.xml

```diff
@@ -69,6 +69,13 @@
   </properties>
 
   <dependencies>
+
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>1.3.2</version>
+    </dependency>
+
```

Comment:
Please change the dependency to 
```
    <dependency>
      <groupId>commons-io</groupId>
      <artifactId>commons-io</artifactId>
      <version>2.5</version>
    </dependency>
```

as it is currently used in the teamengine to avoid dependency conflicts.